docs/content/preview/api/ysql/syntax_resources/grammar_diagrams.md

```diff
@@ -546,8 +546,11 @@ copy_option ::= FORMAT format_name
                 | FORCE_NULL ( column_names )
                 | ENCODING 'encoding_name'
                 | ROWS_PER_TRANSACTION integer
+                | DISABLE_FK_CHECK
+                | REPLACE
+                | SKIP integer
```

Comment:
It will be great to sort them in alphabet order if these options are parallel to each other.